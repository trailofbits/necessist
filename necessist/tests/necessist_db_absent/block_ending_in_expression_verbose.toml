args = [
    "--no-sqlite",
    "--root=fixtures/block_ending_in_expression",
    "--verbose",
]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
