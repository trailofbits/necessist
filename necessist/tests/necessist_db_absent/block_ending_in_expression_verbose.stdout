2 candidates in 1 test in 1 source file
fixtures/block_ending_in_expression/src/lib.rs: dry running
fixtures/block_ending_in_expression/src/lib.rs: mutilating
fixtures/block_ending_in_expression/src/lib.rs:3:5-7:8: `positive(|| {
        let mut n = 0;
        n += 1;
        n
    });` passed
fixtures/block_ending_in_expression/src/lib.rs:5:9-5:16: `n += 1;` failed
